src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -316,6 +316,10 @@ public static function setCommonSpanInfo($source, SpanData $span)
         $span->meta[Tag::COMPONENT] = PDOIntegration::NAME;
         if (\DDTrace\Util\Runtime::getBoolIni("datadog.trace.db_client_split_by_instance")) {
             if (isset($storedConnectionInfo[Tag::TARGET_HOST])) {
+                $mapping = \dd_trace_env_config('DD_SERVICE_MAPPING');
+                if (isset($mapping[$span->service])) {
+                    $span->service = $mapping[$span->service];
+                }
```

Comment:
The only thing bothering me about this piece of code is that, theoretically, it already does this by `Integration::handleInternalSpanServiceName($span, PDOIntegration::NAME);`. I believe this applies to the other integrations as well.

_Not sure that it's necessary_

Reply:
Ah, interesting. It was changed in 0.96.0, I see :-)

Reply:
Removed that part (and left the test), thanks.